build.sbt

```diff
@@ -60,8 +60,8 @@ lazy val backup = project
     version := "1.0-SNAPSHOT"
   )
 
-lazy val registration = project.
-  dependsOn(common)
+lazy val registration = project
+  .dependsOn(common % "test->test;compile->compile")
```

Comment:
This a way of sharing `common/src/test/scala/application/WithPlayApp.scala` trait between several projects (test-scoped)